packages/web3torrent/src/clients/web3torrent-client.ts

```diff
@@ -1,62 +1,29 @@
 import React from 'react';
```

Comment:
Do we need to have a `web3torrent-client`? Seems like we might be able to get rid of this level of indirection and invoke methods of web3torrent-lib directly.

Reply:
I didn't want to change things too much. This PR was really big as is. I could take a look into removing it.